packages/flutter/test/widgets/spell_check_test.dart

```diff
@@ -0,0 +1,318 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+late DefaultSpellCheckSuggestionsHandler defaultSpellCheckSuggestionsHandler;
+late TextStyle composingStyle;
+late TextStyle misspelledTextStyle;
+late String text;
+late String resultsText;
+late TextEditingValue value;
+late bool composingRegionOutOfRange;
+late SpellCheckResults spellCheckResults;
+late TextSpan textSpanTree;
+late TextSpan expectedTextSpanTree;
```

Comment:
Could you declare these variables inside of the tests or is there a reason to declare them here?